scripts/functions/requirements/arch

```diff
@@ -64,12 +69,15 @@ requirements_arch_define()
       requirements_check nodejs
       ;;
     (rbx*head|rubinius*head)
+      undesired_check clang35 llvm35 llvm35-libs
       requirements_check clang llvm llvm-libs patch curl zlib readline autoconf automake diffutils make libtool bison git
       ;;
     (rbx-3*|rubinius-3*)
+      undesired_check clang35 llvm35 llvm35-libs
```

Comment:
I wonder if we need to remove all of it - or only some parts gone would be enough...